db/migrate/20180124104518_add_default_start_time_to_matches.rb

```diff
@@ -1,5 +0,0 @@
-class AddDefaultStartTimeToMatches < ActiveRecord::Migration[5.1]
-  def change
-    change_column_default :matches, :start_time, -> { 'CURRENT_TIMESTAMP' }
```

Comment:
You should not delete an existing migration that has been deployed. Please add a new migration changing the default again or [reverting this one](http://guides.rubyonrails.org/active_record_migrations.html#reverting-previous-migrations)